ws281x_controller.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+from time import sleep
+import rpi_ws281x as neopixel
+
+class WS281XController():
+    """Controller for WS281X LED-strips connected to a Raspberry Pi.
+
+    Attributes:
+        led_count (int): Number of leds in the led strip.
+        led_pin (int): GPIO pin connected to the pixels (18 uses PWM!).
+        led_freq_hz (int): (Optional) LED signal frequency in hertz (usually 800khz).
+        led_dma (int): (Optional) DMA channel to use for generating signal (try 10).
+        led_invert (str): (Optional) True to invert the signal (when using NPN transistor level shift).
+        led_brightness (int): (Optional) LED signal frequency in hertz (usually 800khz).
+        led_channel (int): (Optional) set to '1' for GPIOs 13, 19, 41, 45 or 53..
+        
+    """
+
+    def __init__(self, led_count=1, led_pin=18, led_freq_hz=800000, led_dma=10, led_invert=False, led_brightness=100, led_channel=0):
+        """Connect to Raspberry Pi and initilize the GPIO pins.
+
+        Args:
+            red_pin (int): GPIO pin used for the red LED channel.
+            green_pin (int): GPIO pin used for the green LED channel.
+            blue_pin (int): GPIO pin used for the blue LED channel.
+            host (str): Nme or IP Address of Raspberry Pi.
+
+        """
+        if led_invert == 'False':
+            led_invert = False
+        else:
+            led_invert = True
```

Comment:
This will work with this specific main file, but it might not in other cases. If e.g. the input argument is `led_invert = False` (which it currently is by default), then after these lines `led_invert = True`. 

I think that it would be more intuitive to say that `led_invert` is a boolean and that the user should instead (if needed) do this conversion before calling the constructor. If it turns out that the user inputs something else, then the `neopixel` package hopefully will raise that error.

I.e., you should do this conversion from string to boolean in `main.py` instead.